### PR TITLE
Sonobi Legacy - Add referrer param

### PR DESCRIPTION
**Pull request by JonGoSonobi**
## Type of change

- [x] Feature
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?

## Description of change
Added referrer param to override ref param in bid request.

- apex.prebid@sonobi.com
- [x] official adapter submission

https://github.com/prebid/prebid.github.io/pull/744

### CODE REVIEW

File: modules/sonobiBidAdapter.js [hunk 1]
```diff
@@ -141,6 +141,18 @@ var SonobiAdapter = function SonobiAdapter() {
     return 'desktop';
   }
 
+  function _getReferrer(bids) {
```
Comment (jimee02) [inline modules/sonobiBidAdapter.js]: there's a utility called getTopFrameReferrer in the util lib you can use here.

https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L175

**Comment by JonGoSonobi** [comment]
@ndhimehta any movement on this?

File: test/spec/modules/sonobiBidAdapter_spec.js [hunk 2]
```diff
@@ -416,4 +416,23 @@ describe('Sonobi adapter tests', () => {
       expect(adapter._getPlatform({innerWidth: 1000})).to.equal('desktop')
     })
   })
+  describe('_getPlatform', () => {
```
Comment (snapwich) [inline test/spec/modules/sonobiBidAdapter_spec.js]: Shouldn't this be `_getReferrer`?

**Comment by JonGoSonobi** [comment]
@snapwich yeah that describe block should've been _getReferrer. I fixed it.

**Comment by JonGoSonobi** [comment]
Looking into the failing test. Will fix soon

**Comment by JonGoSonobi** [comment]
@snapwich any idea what the js heap out of memory issue is about in the travis builds?

**Comment by mkendall07** [comment]
restarted the build lets see.

**Comment by JonGoSonobi** [comment]
hmm looks like it was able to pass that issue this time. Any chance we can get this merge in soon? We've got a publisher who paused us until this is resolved and released.

**Pull request merged**